src/express/app.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import { queryParser, cookieParser } from './middlewares';
+import { router } from './routes';
+import { state } from './state';
+
+const app = express();
+
+global.state = state;
```

Comment:
Please, don't set global variables, better use imported data in each file separately